### ISSUE TITLE
pdb default

### ISSUE THREAD

**Issue by kfox1111** [loft-sh/vcluster#839]
### Is your feature request related to a problem?

Not sure if this is a bug fix or a feature request.

Had a user load up a vcluster, install the percona operator and spin up a db cluster. Then expected everything to work properly.

But no pdb's were synced from the over cluster to the under one, so it would break during the rolling upgrades of the nodes of the undercluster.

### Which solution do you suggest?

Defaulting pdb sync to on. Not sure there is a reason not to, and users using operators expect them to "just work" out of the box.

**Comment by matskiv** [comment]
@kfox1111 I think we had hesitations about RBAC - would users that have "edit" permissions in a namespace have permissions to edit PDBs. It is the case with [the default k8s "edit" role](https://github.com/kubernetes/kubernetes/blob/e11f23eb9712c9b4bebf8dd85dcb04441b4fb705/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L314), but I think before merging the change we will double check a few popular distros to make sure.

**Comment by kfox1111** [comment]
Yeah, vanilla seems to allow it. I'd be surprised if many distro's changed from the defaults. The exception might be openshift. Does anyone have the ability to see what it allows by default?
